subscription-service/src/main/resources/application.properties

```diff
@@ -1,23 +1,18 @@
 spring.config.import=classpath:/shared.properties
-
-spring.r2dbc.url = r2dbc:postgresql://localhost/stellio_subscription
-spring.r2dbc.username = stellio
-spring.r2dbc.password = stellio_password
-
+spring.r2dbc.url=r2dbc:postgresql://localhost/stellio_subscription
+spring.r2dbc.username=stellio
+spring.r2dbc.password=stellio_password
 # Required for Flyway to know where the DB is located
-spring.flyway.url = jdbc:postgresql://localhost/stellio_subscription
-spring.flyway.user = ${spring.r2dbc.username}
-spring.flyway.password = ${spring.r2dbc.password}
-
+spring.flyway.url=jdbc:postgresql://localhost/stellio_subscription
+spring.flyway.user=${spring.r2dbc.username}
+spring.flyway.password=${spring.r2dbc.password}
 # Client registration used to get entities from search-service
 spring.security.oauth2.client.registration.keycloak.authorization-grant-type=client_credentials
 spring.security.oauth2.client.registration.keycloak.client-id=client-id
 spring.security.oauth2.client.registration.keycloak.client-secret=client-secret
 spring.security.oauth2.client.provider.keycloak.token-uri=https://my.sso/token
-
 subscription.entity-service-url=http://localhost:8083
-
 # Stellio url used to form the link to get the contexts associated to a notification
 subscription.stellio-url=http://localhost:8080
-
```

Comment:
a lot of spacing changes here

Reply:
It already was added with .contentType(mediaType).
I removed it and add the header in the same way for both